### DISCUSSION TITLE
SFMC Devtools Journey support

### DISCUSSION THREAD

**Discussion by narmadakankuri9** [Accenture/sfmc-devtools#1286] [General]
Hello All!

Is SFMC Dev Tool will supports the complete journey's deployment?

**Comment by JoernBerkefeld** [comment]
hi @narmadakankuri9 - not sure about FULL, but im currently working on a pull request that will ensure complete BASIC support and take it out of beta mode: 
- #1268

with that merged, i would be confident that basic journeys (transactional, single-send and simple multi-step) will function after a cross-BU deployment.

that will cover a journey of any type that sends a single email. next steps will be to ensure **decision splits** dont break. then **entry filters**
- #9 
- #1287

**Reply by JoernBerkefeld** [reply]
overall support progress is tracked here: https://github.com/Accenture/sfmc-devtools/issues/16

**Reply by narmadakankuri9** [reply]
Thank you!
